src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/MarkDuplicatesSparkTester.java

```diff
@@ -9,6 +10,10 @@
  */
 public final class MarkDuplicatesSparkTester extends AbstractMarkDuplicatesTester {
 
+    public MarkDuplicatesSparkTester() {
+        super(true, DuplicateScoringStrategy.ScoringStrategy.TOTAL_MAPPED_REFERENCE_LENGTH);
```

Comment:
Does this mean that we're no longer testing the default duplicates scoring strategy at all?